Getting-Started.adoc

```diff
@@ -1,101 +0,0 @@
-= Sk8s: a FaaS for Kubernetes - Getting Started
```

Comment:
I think we want to keep this guide. The helm charts are working, so this can be adapted (this issue is relevant: https://github.com/projectriff/riff/issues/161). In fact, I think the README should just change to point to either the helm install (for most people) or the manual install (for us and developers who want to operate at that level).

Reply:
Sure, my mistake. I assumed it was meant to go as it hadn't been `s/sk8s/riff`'d.
Seems like it's more "Install via helm" than "Getting started" though?

Reply:
in general for users who aren't going to be working with the framework code, I think the helm install *is* the best getting started experience (the current `README` in `helm-charts` already shows a single command to deploy riff using locally built Docker images

Reply:
When https://github.com/projectriff/riff/pull/182 is ready for merging, it will be modifying this file, so we should leave it as is for now.